src/design/introspection.stanza

```diff
@@ -269,6 +271,21 @@ public defn get-pads-from-port (pt:JITXObject, cxt:JITXObject = self) -> Maybe<T
         val p2p-map = port-to-pads(value(given))
         get-pads-from-port(p2p-map, pt)
 
+doc: \<DOC>
+Get pads for each port and combine them into a single list
+<DOC>
+public defn get-pads-from-ports (pts:Seqable<JITXObject>, cxt:JITXObject = self) -> Seq<JITXObject>:
+  for pt in pts seq-cat:
+    val pds? = get-pads-from-port(pt, cxt)
+    value-or(pds?, [])
+
+doc: \<DOC>
+Convert a Sequence of JITXObjects to LandPatternPad objects.
+<DOC>
+public defn to-lp-pad-seq (objs:Seq<JITXObject>) -> Seq<LandPatternPad> :
+  for obj in objs seq:
+    obj as LandPatternPad
```

Comment:
Only possible change is to match and throw an exception instead of cast -- slightly scary to expose a blind cast via public API

Reply:
OK - perhaps I'm being too cavalier with the `as` operator. Will that `fatal` if it doesn't match correct ?

Reply:
Ya - so I think this function should be in JITX not JSL if anywhere.
In this particular case, I'm confident that this JITXObject is a LandPatternPad. I could put this code in the design but it is going to be duplicated everywhere I use this pattern. 
The other option is to change the `bounds` function to accept a `JITXObject` but that really just moves the problem. When `bounds` tries to inspect a LandPatternPad and it doesn't find one - then an error will be raised there. I think the `jsl/landpatterns/leads/bounds` function was probably written when I was trying for typing - but perhaps I need to re-evaluate that instead of adding this function.